docs/server/upgrading.rst

```diff
@@ -185,12 +185,19 @@ Migrate your database schema
 ----------------------------
 
 Once you have updated your settings you can perform the database schema and
-data upgrade by running. This needs to be done in a few steps:
+data upgrade by running. This is done as follows:
+
+.. note:: The following fake migrations are only required when migrating from
```

Comment:
I am trying to make doubly sure someone doesn't run this in a case where they are not needed, witness what happened the other night.  So I'd prefer to keep unless you have a strong enough wording.